playbooks/roles/ocp-cluster-etcd-operator/tasks/install-kube-burner.yaml

```
@@ -0,0 +1,26 @@
+---
+
+- name: Download kube-burner git repo
+  get_url:
+    url: "https://github.com/kube-burner/kube-burner/releases/download/v1.10.4/kube-burner-V1.10.4-linux-ppc64le.tar.gz"
+    dest: "/tmp/kube-burner-V1.10.4-linux-ppc64le.tar.gz"
+    mode: '0644'  
+
+- name: Extract kube-burner tarball
+  unarchive:
+    src: "/tmp/kube-burner-V1.10.4-linux-ppc64le.tar.gz"
+    dest: "/tmp/"
+    remote_src: yes  # Use to indicate the file is already on the remote machine
+    creates: "/tmp/kube-burner-V1.10.4-linux-ppc64le"  # Prevent extraction if the folder already exists
+
+- name: Verify extracted files
+  command: ls -la /tmp/ | grep kube
```

Please grep full name,
```suggestion
  command: ls -la /tmp/ | grep kube-burner
```